content/docs/arvo/jael/jael.md

```diff
@@ -0,0 +1,25 @@
++++
+title = "Overview"
+weight = 1
+template = "doc.html"
++++
+
+Jael is the networking key infrastructure vane.
+
+The main purpose of Jael is keeping track of [Azimuth](/docs/azimuth/azimuth) related information. For each ship, this consists of its public keys, its key revision number (or `life`), its continuity breach number (or `rift`) (see [Life and Rift](/docs/azimuth/life-and-rift), and who the sponsor of the ship is. It also handles the local ship's private keys, keeps track of galaxy domain prefixes, and performs some boot-related tasks.
```

Comment:
```suggestion
The main purpose of Jael is keeping track of [Azimuth](/docs/azimuth/azimuth) related information. For each ship, this consists of its public keys, its key revision number (or `life`), its continuity breach number (or `rift`) (see [Life and Rift](/docs/azimuth/life-and-rift)), and who the sponsor of the ship is. It also handles the local ship's private keys, keeps track of galaxy domain prefixes, and performs some boot-related tasks.
```

Reply:
I'm with @tinnus-napbus on the source comments. I think they're fine where they are, and ought to be somewhere, and that still seems like the best spot to me.